### ISSUE TITLE
Small needs more contrast

### ISSUE THREAD

**Issue by mgifford** [orderedlist/minimal#79]
The color for small doesn't meet color contrast requirements

```
a small {
    font-size: 11px;
    color: #777;
    margin-top: -0.3em;
    display: block;
}
```

It should be:
 `color: #717171 `

See: https://contrast-ratio.org/#%23777-on-white
